packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -135,9 +121,9 @@ _Component Name_
 _Props/Attrs_
 
 -   `key`: string | number | boolean | undefined
--   `icon`: string - name of the Nimble icon element
+-   `icon`: string | undefined - name of the Nimble icon element. If `undefined`, no icon will be associated with the given `key`.
```

Comment:
I've brought back the `nimble-mapping-text` element so that clients won't have to use `nimble-mapping-icon` even when they never want an icon. I've also added `nimble-mapping-placeholder` to cover some more use cases.